__tests__/docker/install.test.itg.ts

```diff
@@ -14,53 +14,45 @@
  * limitations under the License.
  */
 
-import {describe, test, expect} from '@jest/globals';
+import {describe, expect, it} from '@jest/globals';
 import fs from 'fs';
 import os from 'os';
 import path from 'path';
 
-import {Install, InstallSourceArchive, InstallSourceImage} from '../../src/docker/install';
+import {Install, InstallSource, InstallSourceArchive, InstallSourceImage} from '../../src/docker/install';
 import {Docker} from '../../src/docker/docker';
 import {Exec} from '../../src/exec';
 
+const dockerInstallType = process.env.DOCKER_INSTALL_TYPE || 'archive';
+const dockerInstallVersion = process.env.DOCKER_INSTALL_VERSION || 'latest';
+
 const tmpDir = () => fs.mkdtempSync(path.join(process.env.TEMP || os.tmpdir(), 'docker-install-itg-'));
 
 describe('install', () => {
-  // prettier-ignore
-  test.each([
-    {type: 'image', tag: '27.3.1'} as InstallSourceImage,
-    {type: 'image', tag: 'master'} as InstallSourceImage,
-    {type: 'image', tag: 'latest'} as InstallSourceImage,
-    {type: 'archive', version: 'v26.1.4', channel: 'stable'} as InstallSourceArchive,
-    {type: 'archive', version: 'latest', channel: 'stable'} as InstallSourceArchive,
-  ])(
-    'install docker %s', async (source) => {
```

Comment:
It would still be nice to run these tests outside of CI for development purposes.
WDYT about keeping them, but only use them if the `DOCKER_INSTALL_*` envs are unset?

Reply:
Yes I think that's fine, let me change this

Reply:
Done